settings.gradle

```diff
@@ -70,6 +70,7 @@ includeWithFlags ':it:builders',                    'java'
 includeWithFlags ':it:context-storage',             'java'
 includeWithFlags ':it:dgs',                         'java'
 includeWithFlags ':it:grpcweb',                     'java', 'akka-grpc_2.13'
+includeWithFlags ':it:grpc:ghz',                    'java', 'relocate'
```

Comment:
Why does this need to be relocated?

Reply:
Nope. It does not need to be tested with shaded code.